lib/Driver/ToolChains.cpp

```diff
@@ -1044,6 +1044,24 @@ addLinkSanitizerLibArgsForDarwin(const ArgList &Args,
                      /*AddRPath*/ true, TC);
 }
 
+static void
+addLinkFuzzerLibArgsForDarwin(const ArgList &Args,
+                       ArgStringList &Arguments,
+                       const ToolChain &TC) {
+
+  // libFuzzer requires C++.
+  Arguments.push_back("-lc++");
+
+  // Link libfuzzer.
+  SmallString<128> Dir;
+  getRuntimeLibraryPath(Dir, Args, TC);
+  llvm::sys::path::remove_filename(Dir);
+  llvm::sys::path::append(Dir, "llvm", "libLLVMFuzzer.a");
```

Comment:
@jrose-apple posting back offline discussion: binary packages posted on LLVM.org _do_ include files in `build/lib` directory. Thus whether or not the library is there would be up to package maintainers...